config/api/src/main/java/org/jboss/windup/config/RuleSubset.java

```diff
@@ -190,7 +190,7 @@ public void perform(Rewrite rewrite, EvaluationContext context)
             throw new IllegalArgumentException("Rewrite must be an instanceof GraphRewrite");
 
         /*
-         * Highly optimized loop - for performance reasons. Think before you change this!
+         * Highly optimized loop - for performance reasons. Think before you change this! (lincolnthree)
          */
```

Comment:
Yes, basically. It's optimization isn't so important for us, IMO.

It is not an exact clone, though, so I think it makes more sense here than as an override.